sys/net/gnrc/netif/lorawan/gnrc_netif_lorawan.c

```diff
@@ -345,6 +346,10 @@ static int _set(gnrc_netif_t *netif, const gnrc_netapi_opt_t *opt)
 
     gnrc_netif_acquire(netif);
     switch (opt->opt) {
+        case NETOPT_LORAWAN_ADR:
+            assert(opt->data_len == sizeof(netopt_enable_t));
+            netif->lorawan.adr = *((netopt_enable_t *)opt->data);
```

Comment:
this should rather be something like "gnrc_lorawan_mlme_set(...)" with a MIB_ADR.

Reply:
also, the adr_ack_counter should be reset when ADR is turned on.

Reply:
please remember to add a "get" counterpart.

Reply:
Done